### DISCUSSION TITLE
Blacklist wildcard possible?

### DISCUSSION THREAD

**Discussion by stony2k** [Luligu/matterbridge-zigbee2mqtt#68] [General]
Hi,

I would like to know if a wildcard like * oder % could be implemented? Right know I am excluding devices with their full name and a separate entry for each one.

**Comment by mclever** [comment]
same ask for whitelisting

**Comment by Luligu** [comment]
Hi,
sure is a great idea I will implement it.

Moved like issue with enhancement.

Thanks.
